### PR TITLE
MYST-578 Start using XGO in CI flow

### PR DESCRIPTION
**Pull request by Waldz**

### CODE REVIEW

File: bin/server_package_standalone [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#> bin/server_package_standalone
+
+set -e
+
+BINARY=${BINARY:="build/server/mysterium_server"}
+DIR_BUILD="build/package"
+PACKAGE_FILE="${DIR_BUILD}/`basename -s .exe ${BINARY}`.tar.gz"
+
+# Put all files to package directory (to avoid preserving directory structure)
+DIR_TEMP=`mktemp -d ${DIR_BUILD}/${tempname}.XXXXXX`
```
Comment (ignasbernotas) [inline bin/server_package_standalone]: what's the XXXXXX for?

File: bin/server_package_standalone [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (ignasbernotas) [inline bin/server_package_standalone]: This is identical to bin/client_package_standalone except for binary name. Couldn't we use a single script for both of these?

Comment (ignasbernotas) [reply bin/server_package_standalone]: If not, then could we at least start to tidy up the file structure in bin/ separate the node from client scripts?

File: bin/server_package_standalone (hunk 1 above)
Comment (Waldz) [reply bin/server_package_standalone]: It's template for tmp directory generation

File: bin/server_package_standalone (hunk 2 above)
Comment (Waldz) [reply bin/server_package_standalone]: Soon we will migrate to single binary `mysterium_node`, so I care about it much ATM

File: bin/server_package_standalone [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+###########################################################################
+# Packaging script which creates standalone package.
+#
+# Usage:
+#> bin/server_package_standalone
+
+set -e
+
+BINARY=${BINARY:="build/server/mysterium_server"}
```
Comment (donce) [inline bin/server_package_standalone]: What does this syntax mean?

File: bin/server_package_standalone (hunk 1 above)
Comment (donce) [reply bin/server_package_standalone]: @ignasbernotas more about it here: https://www.mktemp.org/manual.html

Comment (donce) [inline bin/server_package_standalone]: Where does this `tempname` come from? :O

File: bin/server_package_standalone [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+DIR_BUILD="build/package"
+PACKAGE_FILE="${DIR_BUILD}/`basename -s .exe ${BINARY}`.tar.gz"
+
+# Put all files to package directory (to avoid preserving directory structure)
```
Comment (donce) [inline bin/server_package_standalone]: *to avoid preserving* - did you mean *to preserve*?

File: bin/server_package_standalone (hunk 3 above)
Comment (Waldz) [reply bin/server_package_standalone]: Variable can be passed to script via environment variables, but has default value

File: bin/server_package_standalone (hunk 1 above)
Comment (Waldz) [reply bin/server_package_standalone]: `mkdir` templating has some predefined variables

File: bin/server_package_standalone (hunk 4 above)
Comment (Waldz) [reply bin/server_package_standalone]: No. By default, `mkdir` preserves directory structure of global paths

**Pull request merged**